pkg/operator/ceph/csi/spec_test.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package csi
+
+import (
+	"testing"
+
+	"github.com/rook/rook/pkg/operator/test"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStartCSI(t *testing.T) {
+	RBDPluginTemplatePath = "../../../../cluster/examples/kubernetes/ceph/csi/template/rbd/csi-rbdplugin.yaml"
```

Comment:
@rootfs wondering if there is a way to get rid of hardcoding the path with "../../.."... :)

Reply:
that is the relative path to the template file

Reply:
I'd suggest we use the integration tests to validate the yamls instead of a unit test. Then we don't need the relative paths here.

Reply:
another idea is to add a symlink to the files from this folder so the tests can look in the local folder